src/pages/Generate.tsx

```diff
@@ -36,14 +102,93 @@ export const Generate = () => {
     e.preventDefault();
     const input = {
       year: Number(year),
+      term: term,
+      courses: selectedCourses,
+      algorithm1: algorithm1,
+      algorithm2: algorithm2,
     };
+    console.log(input);
     generate({ variables: { input } });
   };
 
+  const handleCourseChange = (
+    courses: MultiValue<CourseOption>,
+    actionMeta: ActionMeta<CourseOption>
+  ) => {
+    if (actionMeta.action === "select-option") {
+      if (actionMeta.option) {
+        const newSelected = [
+          ...selectedCourses,
+          {
+            code: actionMeta.option.value.code,
+            subject: actionMeta.option.value.subject,
+            section: 1,
+          },
+        ];
+        setSelectedCourses(newSelected);
+      } else {
+        console.log("Option Not Defined");
+      }
+    } else if (actionMeta.action === "remove-value") {
+      if (actionMeta.removedValue) {
+        setSelectedCourses(
+          selectedCourses.filter(function (course) {
+            return (
+              course.code !== actionMeta.removedValue.value.code &&
+              course.subject !== actionMeta.removedValue.value.subject
+            );
+          })
+        );
+      } else {
+        console.log("Removed Value Not Defined");
```

Comment:
Same thing here, was this just for debugging or is it an actual error condition?